react/components/atoms/container-openable/container-openable.js

```diff
@@ -213,15 +217,15 @@ export class ContainerOpenable extends PureComponent {
                             onPress={this.onHeaderPress}
                             {...this.props.headerProps}
                         >
-                            <View style={styles.knob} />
+                            {this.props.showKnob ? <View style={styles.knob} /> : null}
                             {this.props.header}
                         </TouchableOpacity>
                     ) : (
                         <View
                             onLayout={event => this._onHeaderLayout(event)}
                             {...this.props.headerProps}
                         >
-                            <View style={styles.knob} />
+                            {this.props.showKnob ? <View style={styles.knob} /> : null}

```

Comment:
Maybe?
```suggestion
                            {this.props.showKnob && <View style={styles.knob} />}

```

Reply:
or `_renderKnob`

Reply:
What is our internal standard? I prefer actually as it currently is because it is more "explicit"

Reply:
> or `_renderKnob`

I dislike because from someone reading it it seems that this will render the knob, when in reality it will first check if it should render the knob and then also render the knob.